src/assets/js/proptypes/index.js

```diff
@@ -25,6 +25,8 @@ export const Event = {
   logo: PropTypes.shape({
     url: PropTypes.string.isRequired
   }).isRequired,
+  thirdPartyText: PropTypes.string.isRequired,
+  organisedBy: PropTypes.string.isRequired,
```

Comment:
These two shouldn't be required. If they don't exist, then we assume TESC.

Reply:
they'll always exist because of the mongoose defaults